src/Network/AWS/Wolf/Act.hs

```diff
@@ -77,13 +77,13 @@ check = maybe (pure False) (liftIO . doesFileExist)
 
 -- | Run a heartbeat.
 --
-startHearbeat :: MonadConf c m => Text -> Int -> Text -> FilePath -> m ()
-startHearbeat queue interval token wd = do
+startHeartbeat :: MonadConf c m => Text -> Int -> Text -> FilePath -> m ()
+startHeartbeat queue interval token wd = do
   traceInfo "heartbeat" mempty
   sd  <- storeDirectory wd
   msd <- metaDirectory sd
   let f = msd </> "heartbeat"
-  writeText f mempty
+  writeText f $ Just ""
```

Comment:
what's the point of this? does a file not get written?

Reply:
If the file isn't getting written, maybe write out a real string like "HEARTBEAT" into the file?

Reply:
just fyi, the `mempty` was for an empty string - "". if you wanted to write heartbeat you'd just write "HEARTBEAT" without a `Just` wrapper.